sdk/common/src/main/java/io/opentelemetry/sdk/common/DaemonThreadFactory.java

```diff
@@ -23,7 +23,7 @@ public DaemonThreadFactory(String namePrefix) {
 
   @Override
   public Thread newThread(Runnable runnable) {
-    Thread t = MoreExecutors.platformThreadFactory().newThread(runnable);
+    Thread t = Executors.defaultThreadFactory().newThread(runnable);
```

Comment:
Previous code has some checks for Google App Engine - no clue what the delta is but this seems reasonable enough.

Reply:
I peeked at this a few weeks ago. It looks like it's old app-engine stuff, so I'm not too worried about it.